src/hooks/useMetricsProcessor.ts

```diff
@@ -398,15 +398,15 @@ export function aggregation(val: number, config: { calculation?: string }): numb
     case Calculations.Apdex:
       data = (val / 10000).toFixed(2);
       break;
+    case Calculations.CPM5D:
+      data = (val / 100000).toFixed(2);
```

Comment:
```suggestion
      data = (val / 100000).toFixed(5);
```

I think it is better to keep 5 digit? @@toffentoffen

Reply:
I think it is better as @wu-sheng suggested. Good catch.
My main reason is to allow converting it back. For instance if we have had only 1 call in a day:
How it will be displayed:
- `(69/100000).toFixed(2) == '0.00'`
- `(69/100000).toFixed(5) ==  '0.00069'
If the user, would like to know how many request they have had:
- `(69/100000).toFixed(2) * 1440` == 0`
- `(69/100000).toFixed(5) * 1440 == 0.9935999999999999` ~ 1 request

Reply:
I think it is better as @wu-sheng suggested. Good catch.
My main reason is to allow converting it back. For instance if we have had only 1 call in a day:
How it will be displayed:
- `(69/100000).toFixed(2) == '0.00'`
- `(69/100000).toFixed(5) ==  '0.00069'
If the user, would like to know how many request they have had:
- `(69/100000).toFixed(2) * 1440` == 0`
- `(69/100000).toFixed(5) * 1440 == 0.9935999999999999` ~ 1 request

Reply:
Although I have to say, that for values `>=100000` it doesn't make sense to display always 5 decimals.
`(100000/100000).toFixed(5) == '1.00000'`
Maybe it should be "dynamic" if value >= 100000 display 2 decimals.

Reply:
Make sense.